frontend/src/components/Module/ModuleDetailsRejectEvents.vue

```diff
@@ -26,49 +26,23 @@
 			</TableTd>
 			<TableTd>
 				{{ moduleRejectEvent.rejectedEvent.__typename }}
+				<InfoTooltip :text="getEventTooltip(moduleRejectEvent.rejectedEvent.__typename!)"/>
 			</TableTd>
 			<TableTd>
-				<div
-					v-if="
-						moduleRejectEvent.rejectedEvent.__typename === 'InvalidInitMethod'
-					"
-				>
-					<div>Init Name:</div>
-					<div>{{ moduleRejectEvent.rejectedEvent.initName }}</div>
-					<div>Module Reference:</div>
-					<div>
-						<ModuleLink
-							:module-reference="moduleRejectEvent.rejectedEvent.moduleRef"
-						/>
+				<div v-if="moduleRejectEvent.rejectedEvent.__typename === 'InvalidInitMethod'">
+					<div>Init name:
+						<InfoTooltip text="Initial entrypoint of the contract."/>
```

Comment:
Use contract name for this. There is no such thing as "initial entrypoint". Conceptually this is a constructor and no instance exists until this succeeds. It's conceputally different from an entrypoint.